kong/plugins/oauth2/migrations/cassandra.lua

```diff
@@ -151,5 +153,23 @@ return {
         end
       end
     end
-  }
+  },
+  {
+    name = "2017-10-19-set_auth_header_name_default",
```

Comment:
This needs to lexically sort, so please add the time in `hhmmss` format to the key (see examples above)

Reply:
nvm, just saw the older ones did this neither.